### ISSUE TITLE
3.1.0: Two maps in one note: first loses markers?

### ISSUE THREAD

**Issue by Moonbase59** [valentine195/obsidian-leaflet-plugin#36]
Upon doing a quick test, I found that when _adding_ a new (test) map to an existing map note, the first map lost all its markers? My mistake?

First map in the note (losing its markers):
```leaflet
id: Friends
markerFolder: People/Friends & Family
markerFolder: +Inbox
height: 500px
lat: 51.133333
long: 10.416667
minZoom: 1
maxZoom: 18
defaultZoom: 5
```

Second (test) map in this note (working):
```leaflet
id: Test
markerFolder: People/Friends & Family
markerTag: #family 
markerTag: #friends, #test 
height: 500px
lat: 51.133333
long: 10.416667
minZoom: 1
maxZoom: 18
defaultZoom: 5
```

_EDIT:_ Deleting the second map and closing/reopening the note (to circumvent caching) didn’t bring back the markers on the first map.

See also: https://forum.obsidian.md/t/new-plugin-obsidian-leaflet-interactive-maps-in-notes/14752/56?u=moonbase59

**Comment by valentine195** [comment]
Okay, I know why the first map lost its markers - it's not due to the second map at all, just an oversight on how the plugin handles _no_ markerTag parameter. Should have a fix out soon.

**Comment by valentine195** [comment]
Could you verify 3.1.1 fixes this?

**Comment by Moonbase59** [comment]
Maybe, but you’re _too_ fast! Fixed in 3.2.1. You’re my current hero!
